graphstorm-processing/.python-version

```diff
@@ -0,0 +1 @@
+3.9
```

Comment:
Do we need this?

Reply:
This is more of a nice-to-have, tells `pyenv` which Python env it should use for this project. I can remove it if you think it clutters the project.